hack/verify-all.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
```

Comment:
@rjsadow curious - do all the scripts under the hack/ directory run as part of the prow CI or needs to be configured somewhere on which one needs to run

Reply:
Right now all of our CI runs by these definitions https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes-sigs/hydrophone. This PR will update our repo and should run `verify-all.sh` script because I've now updated `make verify`. Here's the specific job definition for verify. https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/hydrophone/hydrophone-presubmit.yaml#L47-L68